api/v1alpha1/image_types.go

```diff
@@ -186,6 +186,16 @@ type ImagePropertiesHardware struct {
 	// +kubebuilder:validation:Enum:=e1000;e1000e;ne2k_pci;pcnet;rtl8139;virtio;vmxnet3
 	// +optional
 	VIFModel *string `json:"vifModel,omitempty" glance:"hw_vif_model"`
+
+	// rngModel adds a random-number generator device to the image’s instances.
+	// This image property by itself does not guarantee that a hardware RNG will be used;
+	// it expresses a preference that may or may not be satisfied depending upon Nova configuration.
+	// +optional
+	RngModel *string `json:"rngModel,omitempty" glance:"hw_rng_model"`
```

Comment:
I was going to say we should validate this, but looking at https://docs.openstack.org/glance/latest/admin/useful-image-properties.html we can't do this usefully.

Please can you add a max length of 255 to keep the CEL validation complexity in check (and fix the lint error)? i.e.

```
// +kubebuilder:validation:MaxLength=255
```

Reply:
Thanks for the review! The validation has been added.